### PR TITLE
implement row removal

### PR DESCRIPTION
**Pull request by muskankapoor**
Capable of removing row of the grid
Screen Shot 2021-03-08 at 5 46 35 PM

### CODE REVIEW

**Comment by shikdernyc** [comment]
How did you test this? Can you show your test results

File: js/removeRows.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const removeRows = function(){
+    
+       // remove the above element everytime this function is clicked on 
+       let removeRow = document.getElementsByTagName('tr')[0]; // reference
```
Comment (shikdernyc) [inline js/removeRows.js:4]: What if there are no `tr` elements in the dom.

**Review by shikdernyc — CHANGES_REQUESTED** [review]

**Comment by muskankapoor** [comment]
> How did you test this? Can you show your test results

Hey @shikdernyc just little confused as to what u r asking for. But yes if u press the row remove button then you can see the rows remove in the grid.

**Comment by shikdernyc** [comment]
> > How did you test this? Can you show your test results
> 
> Hey @shikdernyc just little confused as to what u r asking for. But yes if u press the row remove button then you can see the rows remove in the grid.

> "if u press the row remove button then you can see the rows remove in the grid."

Yeah, that's exactly what I meant. Like just written documentation on how you went about testing it or a gif. 

Since we are not using netlify for pull request preview, I would have to go to my computer, pull up this branch, and then run it. Most reviewers won't do that so it's a good idea to put screenshots of ur changes. Or just go ahead and setup netlify if you have the time.

**Comment by muskankapoor** [comment]
Hey, @shikdernyc sure. I m having trouble sharing a video here. 
here sharing the before and after screenshots to prove that my code works.
Please approve the pull request thanks! 
Screen Shot 2021-03-08 at 8 23 46 PM
Screen Shot 2021-03-08 at 8 23 53 PM

**Comment by shikdernyc** [comment]
Looks good to me. There's still another comment I've left in the `js/removeRows.js` file earlier. Can you take a look at that when you get the chance?

File: js/removeRows.js (hunk 1 above)
Comment (muskankapoor) [reply js/removeRows.js:4]: hey, @shikdernyc I thought it was apt to use tr, if you check the populateGrid.js file written by a fellow classmate tr elements is also utilized there, so I think it is fine in this case. 
but if you find any problems in using tr elements then please modify the code as you see. it is in the removeRow branch if you need to modify it.

Comment (shikdernyc) [reply js/removeRows.js:4]: I mean about the edge case of where there's no more <tr/> element in the dom. For example, if someone continuously clicked on "Remove Row", there would be a point where there's no more `tr` in the web page. In which case, wouldn't `document.getElementsByTagName('tr')[0]` result in `OutOfBound` error?

Comment (muskankapoor) [reply js/removeRows.js:4]: Hey @shikdernyc no it should not create problems for the user if they continuously click remove rows. from my branch, the remove row functionality works well. approve this pr so at least we will have the remove row functionality merged into our project.
If I do get a chance (as unfortunately, I m away from my laptop today and a bit of tomorrow) I will add the error messages and or edge cases later in a separate pr

Comment (shikdernyc) [reply js/removeRows.js:4]: Very well. Lmk when you merge in your PR. I'll merge in my PR afterward and deal with any merge conflicts.

**Review by shikdernyc — APPROVED** [review]

File: js/removeRows.js (hunk 1 above)
Comment (muskankapoor) [reply js/removeRows.js:4]: thanks, @shikdernyc yes just merged in my pr

**Pull request merged**